cms/static/js/i18n/ar/djangojs.js

```diff
@@ -269,7 +269,7 @@
     "Already a course team member": "\u0639\u0636\u0648 \u0645\u0633\u0628\u0642 \u0641\u064a \u0641\u0631\u064a\u0642 \u0627\u0644\u0645\u0633\u0627\u0642", 
     "Already a library team member": "\u0639\u0636\u0648 \u0645\u0633\u0628\u064e\u0642 \u0641\u064a \u0641\u0631\u064a\u0642 \u0627\u0644\u0645\u0643\u062a\u0628\u0629", 
     "Already a member": "\u0639\u0636\u0648 \u0645\u0633\u062c\u0651\u064e\u0644 \u0645\u0646 \u0642\u0628\u0644", 
-    "Already have an account?": "\u0647\u0644 \u0644\u062f\u064a\u0643 \u062d\u0633\u0627\u0628 \u0645\u0633\u0628\u0642\u0627\u064b\u061f", 
+    "Already have an account?": "\u0647\u0644 \u0644\u062f\u064a\u0643 \u062d\u0633\u0627\u0628 {platformName} \u0645\u0633\u0628\u0642\u0627\u064b\u061f", 
```

Comment:
Why we needed to add placeholder `{platformName}` in translated string?

Reply:
Edx translations auto-update did this.. I will review..